src/devices/elro_db286a.c

```diff
@@ -0,0 +1,83 @@
+/* Initial doorbell implementation for Elro DB286A devices
+ * 
+ * Note that each device seems to have two id patterns, which alternate
+ * for every other button press.
+ * 
+ * Example pattern: 001101111111011000101010011011001
+ */
+ 
+#include "rtl_433.h"
+#include "pulse_demod.h"
+#include "data.h"
+#include "util.h"
+
+//33 pulses per data pattern
+#define	DB286A_PULSECOUNT		33
+//5*8 = 40 bits, 7 trailing zero bits are encoded, too
+#define DB286A_CODEBYTES		5
+//Hex character count for code:
+//(DB286A_CODEBYTES*8)/4 (8 bits per byte, 4 bits per hex character)
+#define DB286A_CODECHARS		DB286A_CODEBYTES*2
+//Minimum data pattern repetitions (14 is maximum)
+#define	DB286A_MINPATTERN		5
+
+static int doorbell_db286a_callback(bitbuffer_t *bitbuffer) {
+	
+	char time_str[LOCAL_TIME_BUFLEN];
+	data_t *data;
+	bitrow_t *bb = bitbuffer->bb;
+	uint8_t *b = bb[1];
+	unsigned bits = bitbuffer->bits_per_row[1];
+
+	char id_string[DB286A_PULSECOUNT + 1];
+	char *idsp = id_string;
+	
+	unsigned i;
+	
+	if (bits != DB286A_PULSECOUNT) {
+		return 0;
+	}
+	
+	//Get hex string representation of code pattern
+	for (i = 0; i <= DB286A_CODEBYTES; i++) {
+	    idsp += sprintf(idsp, "%02x", b[i]);	
+	}
+	id_string[DB286A_CODECHARS] = '\0';
+	
+	printf("%s", id_string);
```

Comment:
Sorry, I saw it in the last moment and already force pushed the commit again. You were too fast :smile: